codebase/auction-house-primary-sync/src/main/scala/JettyLauncher.scala

```diff
@@ -1,3 +1,3 @@
 import com.virtuslab.base.sync.BaseJettyLauncher
 
-object JettyLauncher extends BaseJettyLauncher
+object JettyLauncher extends BaseJettyLauncher(defaultPort = 8080)
```

Comment:
it's easier to start (especially running from sources) when each service has distinct port